generators/client/templates/angular/src/main/webapp/app/account/password-reset/finish/password-reset-finish.component.ts.ejs

```diff
@@ -52,37 +50,38 @@ export class PasswordResetFinishComponent implements OnInit, AfterViewInit {
     ) {
     }
 
-    ngOnInit() {
+    ngOnInit(): void {
         this.route.queryParams.subscribe((params) => {
             this.key = params['key'];
+            this.keyMissing = !this.key;
         });
-        this.keyMissing = !this.key;
     }
 
-    ngAfterViewInit() {
-        if (this.elementRef.nativeElement.querySelector('#password') != null) {
-          this.renderer.invokeElementMethod(this.elementRef.nativeElement.querySelector('#password'), 'focus', []);
+    ngAfterViewInit(): void {
+        if (this.elementRef.nativeElement.querySelector('#newPassword') != null) {
```

Comment:
I would prefer using a `ViewChild` with a TemplateRef instead of querying by id.. WDYT ? It's not in the scope of this PR so that's up to you if you want to do it :)

Reply:
Good idea, I made this change.